app/_posts/2023-06-23-kuma-2-3-0.md

```diff
@@ -0,0 +1,104 @@
+---
+title: Kuma 2.3 release with MeshTCPRoute, GAMMA support and much more
+description: Kuma 2.3 release with MeshTCPRoute, GAMMA support and much more
+date: 2023-06-23
+headerImage: /assets/images/blog/test-header.jpg
+tags:
+  - Release
+---
+
+We’re excited to announce the release of Kuma 2.3. This new minor release gets us one step closer to releasing targetRef policies as defaults. 
+
+In order to take advantage of the latest and greatest in service mesh, we strongly suggest upgrading to Kuma 2.3. Upgrading is easy through `kumactl` or Helm.
+
+## Notable features:
+
+* 🚀 targetRef policies are now working with `MeshGateway`.
+* 🚀 `MeshTCPRoute` which enables to control traffic routing for TCP services.
+* 🚀 Helm improvements with more secure defaults. 
+* 🚀 Possibility to skip creating default policies when creating a mesh. 
+* 🚀 Performance improvements on Kubernetes. 
+* 🚀 Add hosts selection predicates to `MeshRetry`
+* 🚀 Initial GAMMA routes support. 
+* 🚀 Continuous improvements to the look and feel of the GUI 
+* 🚀 Upgrade to Envoy 1.26
+
+And a lot more! Checkout the full [release notes](https://github.com/kumahq/kuma/releases/tag/2.3.0) to see everything in this release.
+
+### `MeshTCPRoute`
+
+In 2.2.0 we release [MeshHTTPRoute](https://kuma.io/docs/2.2.x/policies/meshhttproute/) which enabled traffic shaping and shifting of HTTP requests.
+In this release we're adding the equivalent [`MeshTCPRoute`](https://kuma.io/docs/2.3.x/policies/meshtcproute/) for TCP traffic.
+
+The usage is very similar to other targetRef policies. For example you can here do a progressive traffic routing of a service from one version to another.
+
+```yaml
+apiVersion: kuma.io/v1alpha1
+kind: MeshTCPRoute
+metadata:
+  name: tcp-route-1
+  namespace: kuma-system
+  labels:
+    kuma.io/mesh: default
+spec:
+  targetRef:
+    kind: MeshService
+    name: frontend_kuma-demo_svc_8080
+  to:
+    - targetRef:
+        kind: MeshService
+        name: backend_kuma-demo_svc_3001
+      rules:
+        - default:
+            backendRefs:
+              - kind: MeshServiceSubset
+                name: backend_kuma-demo_svc_3001
+                tags:
+                  version: "1.0"
+                weight: 90
+              - kind: MeshServiceSubset
+                name: backend_kuma-demo_svc_3001
+                tags:
+                  version: "2.0"
+                weight: 10
+
+```
+
+Note that cross zone routing hasn't been implemented yet. This will be done in the next release ([#6989](https://github.com/kumahq/kuma/issues/6989)).
+
+### `MeshGateway` support in new policy
+
+Up until now it was complicated to use the `targetRef` policies with the builtin gateway (read [#6560](https://github.com/kumahq/kuma/issues/6560) to understand why).
+In the coming release this will be made even simpler with a new targetRef type `MeshGateway` which will enable you to identify a gateway by its name.
```

Comment:
```suggestion
In the upcoming release this will be made even simpler with a new targetRef type `MeshGateway` which will enable you to identify a gateway by its name.
```